Docker/README.md

```diff
@@ -0,0 +1,98 @@
+# X-Road Metrics Docker Containers
```

Comment:
Since this setup spins up a bunch of containers,  it would be good to add minimum hardware requirements to this document. At least the minimum amount of RAM should be mentioned since it's usually the bottleneck.